src/TextEditor/TextEditor.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import React from 'react';
+
+import TextEditor from './TextEditor';
+
+export default {
+  title: 'Common/TextEditor',
+  component: TextEditor,
+} as ComponentMeta<typeof TextEditor>;
+
+const Template: ComponentStory<typeof TextEditor> = (args) => (
+  <TextEditor {...args}>{args.children}</TextEditor>
+);
+
+export const ReadMode = Template.bind({});
+ReadMode.args = {
+  value: 'my text here',
+};
+
+export const EmptyReadMode = Template.bind({});
+EmptyReadMode.args = {
+  value: '',
```

Comment:
Maybe in the Builder, we should show something like "_This document is empty..._" in grey.

Reply:
I move this feature in DocumentItem.